### ISSUE TITLE
Transparent Viewmodels option not working.

### ISSUE THREAD

**Issue by Herwork06** [CriticalFlaw/rayshud-Installer#39]
When you click the transperent view model box and apply its always an error coming up. The error say can not find the file path C:\User\SteamLibrary\steamapps\common\Team Fortress 2\tf\custom\rayshud\cfg\hud.cfg. but you can fix it by making a cfg folder in rayshud and copy hud.cfg from the install folder to cfg.

**Comment by killocode** [comment]
I too am getting this error:
```
---------------------------
Error: Updating Transparent Viewmodels.
---------------------------
An error occurred while applying the viewmodel transparency setting. Could not find a part of the path 'D:\Steam\steamapps\common\Team Fortress 2\tf\custom\rayshud\cfg\hud.cfg'.
---------------------------
```

**Comment by CriticalFlaw** [comment]
This has been fixed in version 4.2.

Thank you for reporting it.
